docs/source/deploy.rst

```diff
@@ -0,0 +1,151 @@
+======
+Deploy
+======
+
+The deployment docs are split up in two parts. First there is the
+general section, which should always be followed. Then there is a cloud
+service provider specific section of which one provider should be chosen.
+
+If you are not sure where to deploy your app, we suggest Heroku. You can test 
+deploying and serving your app without having to enter any credit card details,
+ and with very little prior experience of deployments.
+
+Setup an example project
+========================
+
+1. Create a project directory of notebooks you wish to display. For this
+   tutorial we will clone voila and treat the notebooks folder as our
+   project root.
+
+   .. code:: bash
+
+       git clone git@github.com:QuantStack/voila.git
+       cd voila/notebooks/
+
+2. Add a requirements.txt file to the project directory. This file should 
+   contain all the Python dependencies your voila app needs to run. For this
+   tutorial we will copy the contents of the environment.yml of voila.
+   We omit xleaflet and xeus-cling because these require extra work that is
+   beyond the scope of this guide.
+
+   .. code:: txt
+
+       ipyvolume
+       bqplot
+       scipy
+       ipympl
+       voila==0.1.2
```

Comment:
Maybe change the voila version to the later one

Reply:
Or even unpin it

Reply:
unpinned it